include/h2o.h

```diff
@@ -2173,6 +2169,7 @@ typedef struct st_h2o_proxy_config_vars_t {
     unsigned use_proxy_protocol : 1;
     unsigned tunnel_enabled : 1;
     unsigned connect_proxy_status_enabled : 1;
+    unsigned forward_close_connection : 1;
```

Comment:
This looks good, I would appreciate us moving the doc-comment above as well, even when other properties do not have one.

Reply:
That makes sense.  Done in a1d5cf0.